diagnostics.tf

```diff
@@ -0,0 +1,43 @@
+resource "azurerm_monitor_diagnostic_setting" "diagnostic_settings" {
+  name                       = "app-gw"
```

Comment:
I renamed this so it gets recreated as terraform doesn't match what is in azure currently